Community/Tdarr_Plugin_vdka_Tiered_CPU_CRF_Based_Configurable.js

```diff
@@ -154,14 +267,93 @@ const plugin = (file, librarySettings, inputs, otherArguments) => {
   }
   response.infoLog += '☑File is a video! \n';
 
+  // Check if duration info is filled, if so times it by 0.0166667 to get time in minutes.
+
+  let duration;
+  // If not filled then get duration of stream 0 and do the same.
+  if (typeof file.meta.Duration !== 'undefined') {
+    duration = file.meta.Duration * 0.0166667;
+  } else {
+    duration = file.ffProbeData.streams[0].duration * 0.0166667;
+  }
+
+  // Set up required variables.
+  // Work out currentBitrate using "Bitrate = file size / (number of minutes * .0075)"
+  // Used from here https://blog.frame.io/2017/03/06/calculate-video-bitrates/
+  // eslint-disable-next-line no-bitwise
+
+  const mediaInfoVideoBitrate = ~~(Number(file.mediaInfo.track[1].BitRate) / 1000);
+  const calculatedBitrate = ~~(file.file_size / (duration * 0.0075));
+  let currentBitrate;
+
+  if ((typeof mediaInfoVideoBitrate !== 'undefined') && !isNaN(mediaInfoVideoBitrate)) {
+    currentBitrate = mediaInfoVideoBitrate;
+  } else {
+    currentBitrate = calculatedBitrate;
+  }
+  const sdbitratecutoff = inputs.sdbitrate_cutoff ? inputs.sdbitrate_cutoff : 999999;
+  const hdbitratecutoff = inputs.hdbitrate_cutoff ? inputs.hdbitrate_cutoff : 999999;
+  const fullhdbitratecutoff = inputs.fullhdbitrate_cutoff ? inputs.fullhdbitrate_cutoff : 999999;
+  const uhdbitratecutoff = inputs.uhdbitrate_cutoff ? inputs.uhdbitrate_cutoff : 999999;
```

Comment:
camelCase, so `sdBitrateCutoff`. Plus no need for the ternary since there's defaults.